### DISCUSSION TITLE
Page number style changes on second page of TOC

### DISCUSSION THREAD

**Discussion by flueders** [tud-cd/tudscr#70] [Q&A] [answered]
I want to have roman page numbering for everything before the main content (abstracts, confirmation, TOC, etc.) and afterwards arabic numbering. So I did the following:

```
\maketitle
\pagenumbering{roman}
% ...
\tableofcontents
\pagenumbering{arabic}
```

This almost results in the correct numbering style. However, the second page of the TOC already has the arabic page number ("1"). I use `tudscrreprt` for setting my thesis, but I am not sure if this problem is related to tudscr.
Maybe you can help me here?

**Comment by mrpiggi** [comment, marked as answer]
You have  to clear after the last page of TOC with `\cleardoublepage` or similar before setting new pagenumbering
```
\maketitle
\pagenumbering{roman}
% ...
\tableofcontents
\cleardoublepage
\pagenumbering{arabic}
```

**Reply by flueders** [reply]
Thank you!

**Reply by juaco-montero** [reply]
thanks !
